Cargo.toml

```diff
@@ -31,6 +31,7 @@ displaydoc = { version = "0.2.4", default-features = false }
 flate2 = { version = "1.0.28", default-features = false, optional = true }
 indexmap = "2"
 hmac = { version = "0.12.1", optional = true, features = ["reset"] }
+memchr = "2.7.2"
 num_enum = "0.7.2"
```

Comment:
Caution: the line immediately below this will disappear when https://github.com/zip-rs/zip2/pull/101 is merged.